main.js

```diff
@@ -0,0 +1,26 @@
+    let els = document.querySelectorAll("header nav ul li");
+    for(let el of els){
+        if(el.textContent.includes("All Posts")){
+            el.remove();
+        }
+    }
+
+    let spans = document.querySelectorAll(".featured span");
+    for(let span of spans){
+        if(span.textContent.includes("Read Time")){
+            span.remove();
+        }
+    }
+
+    let posts = document.querySelectorAll("main .posts article");
+    for(let post of posts){
+        if(post.textContent.includes("You -- yes you!")){
+            post.remove();
+        }
+    }
```

Comment:
Interesting way to do this one, nice job! Here's another way to do it in case you're curious:
`posts[posts.length - 1].remove()` <-- grab the last element in the `posts` node list and .remove it.

Reply:
👾